src/Germany/Mews.Fiscalizations.Germany/FiskalyClient.cs

```diff
@@ -56,21 +61,21 @@ public Task<ResponseResult<Tss>> GetTssAsync(AccessToken token, Guid tssId)
             );
         }
 
-        public Task<ResponseResult<Tss>> CreateTssAsync(AccessToken token, TssState state, string description = null)
+        public Task<ResponseResult<CreateTss>> CreateTssAsync(AccessToken token)
         {
-            return Client.ProcessRequestAsync<Dto.CreateTssRequest, Dto.TssResponse, Tss>(
+            return Client.ProcessRequestAsync<object, Dto.CreateTssResponse, CreateTss>(
                 method: HttpMethod.Put,
                 endpoint: $"tss/{Guid.NewGuid()}",
-                request: RequestCreator.CreateTss(state, description),
-                successFunc: response => ModelMapper.MapTss(response),
-                token: token
+                successFunc: response => ModelMapper.MapCreateTss(response),
+                token: token,
+                request: new { }
             );
         }
 
-        public Task<ResponseResult<Tss>> UpdateTssAsync(AccessToken token, Guid tssId, TssState state)
+        public async Task<ResponseResult<Tss>> UpdateTssAsync(AccessToken token, Guid tssId, TssState state)
         {
-            return Client.ProcessRequestAsync<Dto.UpdateTssRequest, Dto.TssResponse, Tss>(
-                method: HttpMethod.Put,
+            return await Client.ProcessRequestAsync<Dto.UpdateTssRequest, Dto.TssResponse, Tss>(
+                method: new HttpMethod("PATCH"),
```

Comment:
`HttpMethod.Patch` doesn't exist?

Reply:
Not in .Net Standard 2.0

if we update to .NET Standard 2.1, then we can do HttpMethod.Patch